public/assets/stylesheets/main.css

```diff
@@ -575,7 +575,8 @@ body {
 
 .modal-hold {
 	width: 320px;
-	height: 900px;
+	height: 100vh;
+	overflow-y: scroll;
 	background: #ffffff;
```

Comment:
Add the missing code to remove scrollbars (to make the ui cleaner)

.modal-hold {
	width: 320px;
	height: 100vh;
	overflow-y: scroll;
	background: #ffffff;
	-ms-overflow-style: none;  /* IE and Edge */
	scrollbar-width: none;  /* Firefox */
}

.modal-hold::-webkit-scrollbar {
	display: none; /*Chrome, Safari and Opera */
}

Reply:
applied